channels/az.m3u

```diff
@@ -21,13 +21,13 @@ https://swow1.socialsmart.tv/aznews/smil:aznews.smil/playlist.m3u8
 http://cbctvlive.flashmediacast.com:1935/CBCTVLive/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="ELTV.az" tvg-name="EL TV" tvg-country="AZ" tvg-language="" tvg-logo="" group-title="",EL TV (432p)
 http://85.132.53.162:1935/live/eltv/playlist.m3u8
-#EXTINF:-1 tvg-id="ITV.az" tvg-name="İTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://www.lyngsat-stream.com/logo/tv/ii/ictimai-tv-az.png" group-title="",İTV
+#EXTINF:-1 tvg-id="ITV.az" tvg-name="İTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://www.lyngsat-stream.com/logo/tv/ii/ictimai-tv-az.png" group-title="",İTV [Geo-blocked]
 https://node19.connect.az:8086/ch1/ch1.hi.m3u8
-#EXTINF:-1 tvg-id="ITV.az" tvg-name="İTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://www.lyngsat-stream.com/logo/tv/ii/ictimai-tv-az.png" group-title="",İTV
+#EXTINF:-1 tvg-id="ITV.az" tvg-name="İTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://www.lyngsat-stream.com/logo/tv/ii/ictimai-tv-az.png" group-title="",İTV [Geo-blocked]
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="ITV.az" tvg-name="İTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://www.lyngsat-stream.com/logo/tv/ii/ictimai-tv-az.png" group-title="",İTV
#EXTVLCOPT:http-referer=https://itv.az/
```

Reply:
referRer. Soesn't work for me

Reply:
Forgot they've written it badly.

Reply:
Didn't help.

Reply:
Weird... could it be that it asks for an user agent too... let me see once again.